doc/policies/authentication.rst

```diff
@@ -187,6 +187,12 @@ Default: *<otp>*
    You should take care, that the *smstext* does not exceed this limit. SMS gateways could
    reject too long messages or the delivery could fail.
 
+.. note:: Some phones expect incoming OTPs as a so called
```

Comment:
I would say "they can handle". Since the phone will also work with a simple OTP value. THe only thing is, that the provide provides an added value, if the text is sent in this format. The default format still works.

Also: Have you checked if a %0A will introduce a linebreak?